README.md

```diff
@@ -28,9 +29,9 @@ Paperize helps game designers make better games by leveraging free web tools to
   - from the Library menu, enable the Google Picker API and choose the previously created API Key
   - from the Library menu, enable the Google Sheets API and choose the previously created API Key
   - Open the OAuth Consent Screen menu again, and add the following scopes:
-    - Google Drive API  ../auth/drive.appdata
-    - Google Drive API  ../auth/drive.file
-    - Google Sheets API ../auth/drive.file
+    - Google Drive API  .../auth/drive.appdata
+    - Google Drive API  .../auth/drive.file
+    - Google Sheets API .../auth/spreadsheets
```

Comment:
I think our goal here is

1. be able to create a spreadsheet (`drive.file`) a
2. be able to read from that spreadsheet (`auth/spreadsheets.readonly` might be sufficient -- so far I used read/write permissions)

https://developers.google.com/sheets/api/guides/authorizing

Reply:
I'll be honest: the Google permissions are confusing to get right.

What we settled on is that we only want permissions that need no special approval from Google. `drive.file` fits this, but I still haven't figured out exactly how it works. Basically, you have full read/write access to any files that you create, but if the user drops a file you DIDN'T create into the Paperize folder, Paperize can't see it. That seems wrong, seems like there should be a way to flag that file as available to Paperize and enable that use case (instead of the janky file uploader I had to write into Paperize itself.)

I spent way too many hours trying to get different things to work, so I'm not looking to spend anymore as what we have works and is quite secure (can't see anything in drive except what we created.) It used to be "full permissions to your entire drive"! 😱

All this said, did you have an actual problem with the permissions as written? Did it fail in a specific way?

Reply:
I ran into an issue with `Google Sheets API ../auth/drive.file`, specifically. I wasn't able to find a permissions for "Google Sheets API" that had this scope name, (`../auth/drive.file`). I suspect it was a mistakenly copied from the scope above. I *think* scope names a globally unique.

<img width="715" alt="Update_selected_scopes_–_APIs___Services_–_paperize-dev_–_Google_API_Console" src="https://user-images.githubusercontent.com/102242/104637378-9235a780-5659-11eb-9959-a817cb80cc42.png">

--

If you have a chance to look at how the production app's OAuth setup is configured, I'm happy to just mirror those scopes in the "getting started" guide.

Reply:
I left this hanging! I'm coming through to get it right now, here's the production scopes:
![image](https://user-images.githubusercontent.com/17697/155848318-9860e0b1-b260-4077-a3c2-382a0db6d759.png)

Not a single sensitive scope, no verification required:
<img width="601" alt="Screen Shot 2022-02-26 at 10 16 21 AM" src="https://user-images.githubusercontent.com/17697/155848436-55937526-5bc1-4153-b6a2-df3a3dbbc590.png">

Going to update the README to only `/auth/drive.file`